nextcloud/content.md

```diff
@@ -123,7 +134,7 @@ The Nextcloud image supports auto configuration via environment variables. You c
 -	`POSTGRES_PASSWORD` Password for the database user using postgres.
 -	`POSTGRES_HOST` Hostname of the database server using postgres.
 
-As an alternative to passing sensitive information via environment variables, `_FILE` may be appended to the previously listed environment variables, causing the initialization script to load the values for those variables from files present in the container. See [Docker secrets](#docker=secrets) section below.
+As an alternative to passing sensitive information via environment variables, `_FILE` may be appended to the previously listed environment variables, causing the initialization script to load the values for those variables from files present in the container. See [Docker secrets](#docker-secrets) section below.
```

Comment:
FWIW, I don't think this link will work correctly on Docker Hub either way :see_no_evil: (see https://github.com/docker/hub-feedback/issues/956)